app/front/scripts/services/visualizations/index.js

```diff
@@ -65,6 +65,13 @@ var availableVisualizations = [
     type: 'pivot-table',
     embed: 'pivottable',
     icon: 'os-icon os-icon-layers'
+  },
+  {
+    id: 'Radar',
+    name: 'Radar Chart',
+    type: 'pivot-table',
```

Comment:
I decided to make it `pivot-table` because it requires same parameters (`rows` and `cols`). But we can make it `sortable-series`, if you think that it will be better